code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -211,6 +211,19 @@
 //Ayy lmao
 
 
+/datum/emote/me/run_emote(mob/user, params, m_type)
+	var/regex/R = new("(\\bd *?(a|@) *?b)(?!bl)","gi") // dab prevention, very thorough, but does not include biodegradable or dabbles for example
```

Comment:
Make sure it doesn't catch these:
https://www.wordfind.com/contains/dab/

The ?!bl is quite clever because that knocks off a ton of words, but you gotta be thorough